ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -61,6 +61,7 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Optional:    true,
 				Computed:    true,
 				Description: "set to true to enable migration of the PI instance",
+				Deprecated:  "This field is deprecated, use pin_policy instead",
```

Comment:
same as above --  Since it is deprecated , we might as well remove it.

Reply:
Removed deprecated fields.